src/UI/Modal/Modal.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+import classes from './Modal.module.css';
+
+const Backdrop = ({ onClose }) => {
+  return <div className={classes.backdrop} onClick={onClose} />;
+};
+
+const ModalOverlay = ({ children }) => {
+  return (
+    <div className={classes.modal}>
+      <div className={classes.content}>{children}</div>
+    </div>
+  );
+};
```

Comment:
as these are components on their own better to have them on separate files. You can have them inside Modal folder like /Moda/MoralOverlay